### PR TITLE
Fix Bug 1457580 - Update strings-import to get strings from l10n-central

### PR DESCRIPTION
**Pull request by Mardak**
Waiting for Pike's feedback before review.

### CODE REVIEW

File: bin/strings-import.js [hunk 1]
```diff
@@ -2,74 +2,78 @@
...
 
 /* eslint-disable no-console */
+const fetch = require("node-fetch");
+
+/* globals cd, ls, mkdir, rm, ShellString */
+require("shelljs/global");
 
 const DEFAULT_LOCALE = "en-US";
+const L10N_CENTRAL = "https://hg.mozilla.org/l10n-central";
+const PROPERTIES_PATH = "raw-file/tip/browser/extensions/activity-stream/activity-stream.properties";
```
Comment (flodolo) [inline bin/strings-import.js]: I think you want `default` here, instead of `tip`.

File: bin/strings-import.js [hunk 2]
```diff
@@ -2,74 +2,78 @@
...
-/* globals cd, cp, ls, mkdir, rm */
-require("shelljs/global");
+// Get all the locales in l10n-central
+async function getLocales() {
+  console.log(`Getting locales from ${L10N_CENTRAL}`);
 
-const args = process.argv.slice(2);
-if (args.length < 1) {
-  throw Error("Please provide the path to the target strings repository");
+  const html = await (await fetch(L10N_CENTRAL)).text();
```
Comment (flodolo) [inline bin/strings-import.js]: You can use `https://hg.mozilla.org/l10n-central/?style=raw`

Comment (Pike) [reply bin/strings-import.js]: ... or json, depending on your preference.

Comment (flodolo) [reply bin/strings-import.js]: I didn't know about https://hg.mozilla.org/l10n-central/?style=json

That makes things a lot less hacky

File: bin/strings-import.js [hunk 3]
```diff
@@ -2,74 +2,74 @@
...
 
-// Use the first script argument as the source of localized strings
-const [stringsPath] = args;
-const stringsRepo = require("simple-git")(stringsPath);
+// Save the properties file to the locale's directory
+async function saveProperties(locale) {
+  // Only save a file if the repository has the file
+  const url = `${L10N_CENTRAL}/${locale}/${PROPERTIES_PATH}`;
+  const response = await fetch(url);
+  if (response.status !== 200) {
```
Comment (k88hudson) [inline bin/strings-import.js]: `if (!response.ok)` might be a little less restrictive

Comment (Mardak) [reply bin/strings-import.js]: ok? ok!? We're not ok! Must be 200!!! ok… ok.

Comment (Mardak) [reply bin/strings-import.js]: Updated ;)

**Pull request merged**